src/index.js

```diff
@@ -0,0 +1,204 @@
+};
+
+const getTasksFromLocStore = () => {
+  const data = JSON.parse(localStorage.getItem('list'));
+  // eslint-disable-next-line array-callback-return
+  data.map((tasks) => {
+    myArray.push(tasks);
+    const todoContainer = document.createElement('div');
+    todoContainer.className = 'todoContainer';
+    todoContainer.innerHTML += `
+        <input type='checkbox' class='checkbox'>
+        <span>${tasks.description}</span>
+        <i class="uil uil-ellipsis-v"></i>
+        <i class="uil uil-trash-alt"></i>
+    `;
+    todosParent.appendChild(todoContainer);
+
+    const editIcons = document.querySelectorAll('.uil-ellipsis-v');
+    editIcons.forEach((task) => {
+      task.addEventListener('click', () => {
+        editTodo(todoContainer, task.previousElementSibling);
+        task.parentElement.classList.add('completedTask');
+      });
+    });
+  });
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((task) => {
+    task.addEventListener('click', () => {
+      task.parentElement.classList.toggle('completedTask');
+      task.nextElementSibling.classList.toggle('checkTodo');
+      task.parentElement.lastElementChild.classList.toggle('trash-active');
+      task.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      sendToLocalAgain();
+    });
+  });
+
+  const removeIcons = document.querySelectorAll('.uil-trash-alt');
+  removeIcons.forEach((task) => {
+    task.addEventListener('click', () => {
+      deleteTask(task.parentElement);
+    });
+  });
+
+  localStorage.setItem('list', JSON.stringify(myArray));
+};
+
+// Add Tasks
+const addTask = (todoValue) => {
+  const todoContainer = document.createElement('div');
+  todoContainer.className = 'todoContainer';
+  todoContainer.innerHTML += `
+        <input type='checkbox' class='checkbox'>
+        <span>${todoValue}</span>
+        <i class="uil uil-ellipsis-v"></i>
+        <i class="uil uil-trash-alt"></i>
+    `;
+  todosParent.appendChild(todoContainer);
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((task) => {
+    task.addEventListener('click', () => {
+      task.parentElement.classList.toggle('completedTask');
+      task.nextElementSibling.classList.toggle('checkTodo');
+      task.parentElement.lastElementChild.classList.toggle('trash-active');
+      task.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      sendToLocalAgain();
+    });
+  });
+
+  // Send Added Tasks  to Local Storage
+
+  // New object instance
+  const object = new MyObject(todoValue, false, checkbox.length - 1);
+  myArray.push(object);
+  localStorage.setItem('list', JSON.stringify(myArray));
+
+  // Edit Tasks
+  const editIcons = document.querySelectorAll('.uil-ellipsis-v');
+  editIcons.forEach((task) => {
+    task.addEventListener('click', () => {
+      editTodo(todoContainer, task.previousElementSibling);
+    });
+  });
+};
```

Comment:
- [ ] When **I edit a task description and click enter, the changes are not saved in the local storage while they are supposed to be reflected there in the local storage**. Please try to correct it 😺